src/domain/common/notifications/notifications.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Challenge } from '@domain/challenge/challenge/challenge.entity';
+import { Ecoverse } from '@domain/challenge/ecoverse/ecoverse.entity';
+import { Opportunity } from '@domain/collaboration';
+import { Discussion } from '@domain/communication/discussion/discussion.entity';
+import { Community } from '@domain/community/community';
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { NotificationsPayloadBuilder } from './notifications.payload.builder';
+
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([Ecoverse]),
+    TypeOrmModule.forFeature([Challenge]),
+    TypeOrmModule.forFeature([Opportunity]),
+    TypeOrmModule.forFeature([Community]),
+    TypeOrmModule.forFeature([Discussion]),
+  ],
+  providers: [NotificationsPayloadBuilder],
+  exports: [NotificationsPayloadBuilder],
+})
+export class NotificationsModule {}
```

Comment:
This whole module belongs I think under services, not directly in the domain model.

Reply:
Actually, I was thinking of removing the module altogether and adding the builder to the microservices module, as it logically belongs there (I think). Then we'll have all the logic required to send an event in one module and will import only one module instead of two (we always import the two modules (Microservices / Notifications) where we send a notification.

Reply:
Interesting, so basically the assertion is that Microservies **is** notifications. Are there other types of microservices that we might want to use in the future? I can definitely see the microservices as being part of the core, but building of notification payloads for me just feels different.